common/src/config/config.ts

```diff
@@ -1620,6 +1628,98 @@ const GAME_PT_CONFIGS: GamePTConfigs = {
 		tierlists: [],
 		tierlistDescriptions: {},
 	},
+	"maimaidx:Single": {
+		idString: "maimaidx:Single",
+
+		percentMax: 101,
+
+		defaultScoreRatingAlg: "rate",
+		defaultSessionRatingAlg: "rate",
+		defaultProfileRatingAlg: "naiveRate",
+
+		scoreRatingAlgs: ["rate"],
+		sessionRatingAlgs: ["rate"],
+		profileRatingAlgs: ["naiveRate", "rate"],
+
+		scoreRatingAlgDescriptions: {
+			rate: "Rating as it's implemented in game.",
+		},
+		profileRatingAlgDescriptions: {
+			naiveRate: "A naive rating algorithm that just averages your 50 best scores.",
```

Comment:
```suggestion
			naiveRate: "A naive rating algorithm that just sums your 50 best scores.",
```

Reply:
Actually this typo apparently also exists in the wacca description

Reply:
yeah i just kinda copied and pasted wacca's, since it matches maimai's system and i didn't really look back at it